src/features/SmartPackage/SmartPackage.tsx

```diff
@@ -72,42 +72,57 @@ export const SmartPackage = () => {
         position: 'relative',
       }}
     >
-      <Stack sx={{ width: '100%', maxWidth: 400, mt: 8.5 }} spacing={2}>
-        <Typography variant='h5' sx={{ fontWeight: 'medium', textAlign: 'center' }}>
-          Smart Package Locker
-        </Typography>
```

Comment:
Container should match the other features like Guest Access so the UI is consistent
```
<Container
      sx={{
        height: '600px',
        marginTop: '60px',
      }}
    >
      <Paper
        sx={{
          padding: '20px',
          borderRadius: '10px',
        }}
      >
```
Otherwise it doesn't have a set width and appears like this when there are no packages
<img width="1207" alt="Screenshot 2025-03-24 at 9 42 25 PM" src="https://github.com/user-attachments/assets/0a7bec2e-007b-4111-b3f6-02aa2bd457c2" />